binding/nodejs/README.md

```diff
@@ -28,59 +13,51 @@ yarn add @picovoice/pvrecorder-node
 
 ## Usage
 
-Getting the list of input devices does not require an instance:
+To start recording initialize the instance and run `start`:
 
 ```javascript
 const { PvRecorder } = require("@picovoice/pvrecorder-node");
 
-const devices = PvRecorder.getAudioDevices();
+const recorder = new PvRecorder(/*sets to default device*/-1, /*frame length*/ 512);
```

Comment:
The formatting on this line is a little odd and device index can be removed since we talk about that later